docs/_v1_6_11/5_sertifikater.md

```diff
@@ -6,26 +6,32 @@ description: Installere sertifikater for signering og kryptering av post
 isHome: false
 ---
 
-For å sende sikker digital post trenger du å installere sertifikater. Grunnen til at vi ønsker å installere de er først og fremst sikkerhet. Alle sertifikater har en unik identifikator som kalles thumbprint. Hvis du ikke ønsker å håndtere selv i koden hvordan sertifikatene skal lastes, så kan du følge guiden under, steg for steg. Til slutt gjennomgås det hvordan du kan finne thumbprint til de installerte sertifikatene.
+Som bruker av dette biblioteket er du en Databehandler som har ansvar for sending av meldinger. For å gjøre dette trenger du et sertifikat for å kunne autentisere deg mot Meldingsformidleren. Du kan lese mer om aktørene [hos begrep.difi.no](http://begrep.difi.no/SikkerDigitalPost/forretningslag/Aktorer). Dette bør installeres på maskinen. Grunnen til at vi ønsker å installere det er først og fremst sikkerhet.
```

Comment:
Hva med _"Sertifikatet må installeres på webserveren(e) din(e)"_?

Den siste setningen med " ... først og fremst sikkerhet" blir litt rar, de må jo ha sertifikat for å kunne autentisere seg?